prog4/jcdriver/main.go

```diff
@@ -7,9 +7,30 @@ import (
 	"time"
 
 	"github.com/riking/joycon/prog4/consoleiface"
+	"github.com/riking/joycon/prog4/jcpc"
+	"flag"
```

Comment:
run `goimports -w` as the formatter - "flag" belongs with the other stdlib imports